### PR TITLE
PR: Add loghub recipe

### PR DESCRIPTION
**Pull request by goanpeca**

### CODE REVIEW

File: recipes/loghub/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+
+test:
+  imports:
+    - loghub
```
Comment (ccordoba12) [inline recipes/loghub/meta.yaml]: This test is not that important because this is not a library. I think it's better

    commads:
        - loghub --help

Comment (goanpeca) [inline recipes/loghub/meta.yaml]: :+1:

**Comment by jakirkham** [comment]
Thanks guys. 😄

**Comment by ccordoba12** [comment]
Thanks for the quick merge @jakirkham!

**Pull request merged**